src/app/content/search/hooks/index.ts

```diff
@@ -65,28 +65,29 @@ export const receiveSearchHook: ActionHookBody<typeof receiveSearchResults> = (s
     services.dispatch(selectSearchResult(selectedResult));
   }
 
-  let navigation = selectNavigation.match(services.getState());
   const targetPageId = selectedResult ? selectedResult.result.source.pageId : currentPage ? currentPage.id : null;
 
-  if (targetPageId) {
-    const targetPage = assertDefined(
-      findArchiveTreeNodeById(book.tree, targetPageId),
-      'search result pointed to page that wasn\'t in book'
-    );
-
-    // tslint:disable-next-line: no-console
-    console.log('targetPage.id: ', targetPage.id);
-
-    navigation = {
-      params: getBookPageUrlAndParams(book, targetPage).params,
-      route: content,
-      state : {
-        bookUid: book.id,
-        bookVersion: book.version,
-        pageUid: stripIdVersion(targetPage.id),
-      },
-    };
-  }
+  const createUpdatedNavigation = () => {
```

Comment:
This function will be defined every time when you call `receiveSearchHook`
If you extract this function to another file then it will be defined only once + logic in `receiveSearchHook` will become more straightforward

Reply:
You are right. I'm wondering if I should create a separate file in `search/hooks/utils.ts` or just use the existing `search/utils.ts` file, because I'm not sure if this util-function will used anywhere else.

Reply:
I guess `search/utils.ts` is fine